### PR TITLE
Complete Week 3, Step3: JSON

### PR DESCRIPTION
**Pull request by danieldortega77**
Add button to homepage that fetches comments from the Java applet

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -11,3 +11,24 @@
...
+  const comments = await response.json();
+  const commentsElement = document.getElementById('comments-container');
+  commentsElement.innerHTML = '';
+
+  for (var comment of comments) {
+    commentsElement.appendChild(createListElement(comment))
+  }
+}
+
+/** Creates an <li> element containing text. */
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: Tiny nit: it's probably better to have a consistent comment style where you use the blank lines, like:
```
/**
 * Creates an <li> element containing text.
 */
```

**Review by dborkan — APPROVED** [review]
👍 - LGTM (looks good to me)

**Review by dslucas — APPROVED** [review]
👍

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (danieldortega77) [reply portfolio/src/main/webapp/script.js]: Got it!

**Pull request merged**